migration/migrating_3_4/configuring-replication-repository.adoc

```diff
@@ -10,12 +10,20 @@ You must configure an object storage to use as a replication repository. The Clu
 
 The following storage providers are supported:
 
-* Generic S3 object storage, for example, NooBaa, Minio, Ceph S3
-* xref:../../migration/migrating_3_4/configuring-replication-repository.adoc#migration-configuring-aws-s3_{context}[AWS S3]
-* xref:../../migration/migrating_3_4/configuring-replication-repository.adoc#migration-configuring-gcp_{context}[Google Cloud Provider]
-* xref:../../migration/migrating_3_4/configuring-replication-repository.adoc#migration-configuring-azure_{context}[Microsoft Azure]
+* Generic S3 object storage, for example, Minio or Ceph S3
+* Multi-Cloud Object Gateway
```

Comment:
Suggest putting the acronyms after all of these, because later on you just use MCG (and I don't think that's a very well known acronym and it might be good to reiterate here as well).

Reply:
> Suggest putting the acronyms after all of these

OK

> later on you just use MCG

Oops -- I thought I remembered not to do that. OK, will check. thanks.